.drone.yml

```diff
@@ -41,15 +41,25 @@ pipeline:
       - cp -R . /tmp/3/ && cd /tmp/3/
       - ./project/scripts/sbt dotty-optimised/testAll
 
-  test_sbt:
+  test_sbt_1of2:
     group: test
     image: lampepfl/dotty:2017-10-20
     commands:
       - cp -R . /tmp/4/ && cd /tmp/4/
-      - ./project/scripts/sbt sbt-dotty/scripted
-    when:
+      - ./project/scripts/sbt "sbt-dotty/scripted compilerReporter/* discovery/* sbt-dotty/* source-dependencies/*1of2"
+    # when:
       # sbt scripted tests are slow and don't run on PRs
-      event: [ push, tag, deployment ]
+      # event: [ push, tag, deployment ]
+
+  test_sbt_2of2:
+    group: test
+    image: lampepfl/dotty:2017-10-20
+    commands:
+      - cp -R . /tmp/5/ && cd /tmp/5/
+      - ./project/scripts/sbt "sbt-dotty/scripted source-dependencies/*2of2"
```

Comment:
It would be great if we did not introduce this huge regression (3x) in the time needed to run the scripted tests.
@Duhemm says it is because the bridge is recompiled for each scripted test. @smarter do you have an idea how this could be fixed?

Reply:
Complain to the sbt people: https://github.com/sbt/sbt/issues/3469 Maybe ask someone who has a lightbend subscription? :)

Reply:
Alternatively, we could try copy-pasting the custom implementation of scripted used by zinc, it even has parallelism and fancy colors contributed by @jvican :P. https://github.com/sbt/zinc/blob/1.x/project/Scripted.scala https://github.com/sbt/zinc/tree/1.x/internal/zinc-scripted/src/test/scala/sbt/internal/inc